techmatters/flex-plugins
# Conferencing: continue 3-way call after 1 person hangs up

This PR is related to https://github.com/techmatters/serverless/pull/474.

## Description
This PR adds logic to handle 3+ participants in a conference, not causing the conference to end if one of them hangs.

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1960)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts

### Verification steps
- Make sure the above branch of Serverless is deployed.
- `npm run dev` to start local Flex.
- Start a new call contact.
- Accept with the Flex instance.
- Start a conference with a 3rd participant.
- Confirm that if there are more than 2 parties in conference, one hanging won't end the conference.
- Confirm that if there are 2 parties, one hanging will end the conference.
